.github/workflows/stale.yml

```diff
@@ -1,18 +0,0 @@
-name: Mark stale issues and pull requests
```

Comment:
I believe you should make a separate PR to remove the Stale workflow. If you need any help on this, let us know! 😄